homework/object-oriented-programming/src/main/java/com/cbfacademy/shapes/Shapes.java

```diff
@@ -0,0 +1,31 @@
+package com.cbfacademy.shapes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Shapes {
+
+    public void computeArea(List<Enclosure> enclosure){
```

Comment:
This variable represents a collection, so should be named with a plural, i.e. `enclosures`